.azure-pipelines/env.yml

```diff
@@ -181,15 +181,36 @@ jobs:
       set -e
 
       PUBLISH_GITHUB_RELEASE=false
-      PUBLISH_NETLIFY=false
-      if [[ -n "$POSTSUBMIT" && "$NOSYNC" != true ]]; then
-          PUBLISH_NETLIFY=true
-      fi
-      if [[ -n "$POSTSUBMIT" && "$NOSYNC" != true && "$ISSTABLEBRANCH" == True && "$(state.isdev)" == false ]]; then
-          PUBLISH_GITHUB_RELEASE=true
+      PUBLISH_DOCKERHUB=false
+      PUBLISH_DOCS=false
+      PUBLISH_DOCS_LATEST=false
+      PUBLISH_DOCS_RELEASE=false
+
+      if [[ "$ISSTABLEBRANCH" == True && -n "$POSTSUBMIT" && "$NOSYNC" != true ]]; then
```

Comment:
still be confused by these `True` and `true`, orz.

Reply:
one is set by azp, the other is set as a string in the azp lib